ansible_collections/arista/avd/roles/eos_validate_state/tasks/main.yml

```diff
@@ -3,9 +3,8 @@
 # that can be found in the LICENSE file.
 ---
```

Comment:
I beleive we would want to remove the tags on the "Create validation reports" lines 65-66.

Reply:
yes. I did not look at this, since i thought it was covered by the previous changes to tags for this role.

Reply:
removed!